### PR TITLE
created play_sessions_controller with user action

### PR DESCRIPTION
**Pull request by diego-unternaehrer**
Please review my code with a close eye to the my_play_sessions action and if the routes still make sense like this.

### CODE REVIEW

File: app/controllers/play_sessions_controller.rb [hunk 1]
```diff
@@ -1,2 +1,69 @@
...
+    end
+  end
+
+  def edit
+  end
+
+  def update
+    @play_session.update(play_session_params)
+    if @play_session.save
+      redirect_to my_play_sessions_path
```
Comment (raquelgraca) [inline app/controllers/play_sessions_controller.rb]: redirect_to my_play_sessions_play_sessions_path

File: app/controllers/play_sessions_controller.rb [hunk 2]
```diff
@@ -1,2 +1,69 @@
...
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    @play_session.destroy
+    redirect_to my_play_sessions_path
```
Comment (raquelgraca) [inline app/controllers/play_sessions_controller.rb]: redirect_to my_play_sessions_play_sessions_path

File: app/controllers/play_sessions_controller.rb (hunk 1 above)
Comment (diego-unternaehrer) [reply app/controllers/play_sessions_controller.rb]: done

File: app/controllers/play_sessions_controller.rb (hunk 2 above)
Comment (diego-unternaehrer) [reply app/controllers/play_sessions_controller.rb]: done

**Pull request merged**